cpp/tests/strings/combine/join_strings_tests.cpp

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
The code in this file is not part of this PR. I just extract one large cpp test file into multiple files.